### ISSUE TITLE
Can't find notebook_plotter tutorial notebook in ciaox

### ISSUE THREAD

**Issue by dprincipe2** [cxcsds/ciao-contrib#908]
Hi @DougBurke, @kglotfelty, @hamogu,

I see that the [notebook_plotter tool](https://github.com/cxcsds/ciao-contrib/blob/main/sherpa_contrib/notebook_plotter.py) I made last year is properly in ciaox but [the notebook tutorial](https://github.com/cxcsds/ciao-contrib/blob/main/share/sherpa/notebooks/notebook_plotter_tutorial.ipynb) associated with tool is not. Should I be seeing it somewhere? My 'ciaover -v' return is pasted below.

If you recall from [this PR](https://github.com/cxcsds/ciao-contrib/pull/878#issuecomment-2078219486), it didn't properly make it into the last ciao-contrib release either so I just wanted to double check it makes it into this one. Thanks!

The current environment is configured for:
  CIAO        : CIAO 4.17.0 Friday, September 20, 2024
  Contrib     : Package release 2  Tuesday, August 20, 2024
  bindir      : /opt/anaconda3/envs/ciaox417_sep27/bin
  Python path : /opt/anaconda3/envs/ciaox417_sep27/bin
  CALDB       : 4.11.3

**Comment by nplee** [comment]
@dprincipe2 I don't think the tutorial made it into the latest contrib package release, but the packing/installation problem we were having should be fixed by PR #902...

You should be able to pip install it with the fix in your activated CIAO environment by doing something like:

```
% git clone https://github.com/cxcsds/ciao-contrib.git contrib.delme
% cd contrib.delme
% pip install . --ignore-installed

% ls $ASCDS_INSTALL/share/sherpa/notebooks/
notebook_plotter_tutorial.ipynb
```

**Comment by DougBurke** [comment]
I'm going to assume that this is now fixed but please holler if it's not (it appears to be for me but my environment is not 100% clean ...).
